### PR TITLE
Fixes for some some typos and grammar errors

### PR DESCRIPTION
**Pull request by jroyvde**
Fixing some typos and grammar mistakes I noticed during my playthrough.

### CODE REVIEW

File: Update/hima_003_04.txt [hunk 1]
```diff
@@ -839,7 +839,7 @@ void main()
 //　去るのみの自分に何か危害が加えられる可能性は、常識的に考えてとても低い＠
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "　去るのみの自分に何か危害が加えられる可能性は、常識的に考えてとても低い。",
-		   NULL, "With the only thing left for me to do was leaving, the chance that I would come to harm was logically fairly low.", GetGlobalFlag(GLinemodeSp));
+		   NULL, "With the only thing left for me to do being to leave, the chance that I would come to harm was logically fairly low.", GetGlobalFlag(GLinemodeSp));
```
Comment (drojf) [inline Update/hima_003_04.txt:842]: Would 

"With leaving being the only thing left for me to do,"

work here?

**Review by drojf — COMMENTED** [review]
There's just one part where it may be worth re-wording the sentence fragment rather than trying to fix it. I'm not very good at this stuff myself, let me know if it seems OK to change

File: Update/hima_003_04.txt (hunk 1 above)
Comment (drojf) [inline Update/hima_003_04.txt]:
```suggestion
		   NULL, "With leaving being the only thing left for me to do, the chance that I would come to harm was logically fairly low.", GetGlobalFlag(GLinemodeSp));
```

Would re-wording the sentence like this work instead?

Comment (jroyvde) [reply Update/hima_003_04.txt]: Yep, I do think this flows a bit better! :)

**Pull request merged**